pkg/profilecreator/profilecreator.go

```diff
@@ -374,7 +374,7 @@ func (ghwHandler GHWHandler) getCPUsSplitAcrossNUMA(reservedCPUCount int, htEnab
 		if max%2 != 0 && htEnabled {
 			return reservedCPUSet.Result(), isolatedCPUSet, fmt.Errorf("can't allocate odd number of CPUs from a NUMA Node")
 		}
-		addCoresToCPUSet(reservedCPUSet, max, node.Cores)
+		addCoresToCPUSet(*reservedCPUSet, max, node.Cores)
```

Comment:
I think a rebase is needed to make sure we are aligned with the latest and remove these diffs

Reply:
Sounds good, I rebased but that change seems to be coming from the updated kubernetes pkg, the builder return type is a [pointer now](https://github.com/eggfoobar/performance-addon-operators/blob/library_go_bump/vendor/k8s.io/kubernetes/pkg/kubelet/cm/cpuset/cpuset.go#L39)

Reply:
I think you'll also need to examine the functions such as 
https://github.com/openshift-kni/performance-addon-operators/blob/4a65bd7f3a80a688b78be1eed1c47a7410c44edf/pkg/profilecreator/profilecreator.go#L356
This should start by looking at the failing tests around cpusets under ci/prow/ci job

Reply:
the rebase and the related fixes should be addressed on a separate PR. Do we need kube >= 1.23?